### PR TITLE
What's New update for 0.30.0

### PR DESCRIPTION
**Pull request by pshipton**
Fixes https://github.com/eclipse-openj9/openj9-website/issues/300

### CODE REVIEW

File: news-page-content/latestRelease.md [hunk 1]
```diff
@@ -22,16 +22,20 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
...
+### Eclipse OpenJ9 version 0.30.0 released
 
-We're pleased to announce the availability of Eclipse OpenJ9 v0.29.0 and v0.29.1.
+We're pleased to announce the availability of Eclipse OpenJ9 v0.30.0.
 
-- The 0.29.0 release supports OpenJDK 8 and 11. The 0.29.1 release supports OpenJDK 17. For more information about supported platforms and OpenJDK versions, see [Supported environments](https://www.eclipse.org/openj9/docs/openj9_support/).
+- The 0.30.0 release supports OpenJDK 8, 11 and 17. For more information about supported platforms and OpenJDK versions, see [Supported environments](https://www.eclipse.org/openj9/docs/openj9_support/).
 
-- In this release, Linux AArch64 is a fully supported, production-ready target. JITServer technology is also fully supported on Linux on x86 and Linux on IBM Power systems (64-bit only).
+- In this release, obsolete shared cache files created by older releases should be removed, ignored options are listed in the javacore, and a new option `-XX:[+|-]EnsureHashed` is added.
```
Comment (zl-wang) [inline news-page-content/latestRelease.md:31]: does it mean "should be removed automatically"?  or the "user" should remove it explicitly?

Comment (mpirvu) [reply news-page-content/latestRelease.md:31]: I agree that the phrase is a bit ambiguous.

Comment (pshipton) [reply news-page-content/latestRelease.md]: Updated to clarify it.

**Review by zl-wang — APPROVED** [review]

File: news-page-content/latestRelease.md [hunk 2]
```diff
@@ -22,16 +22,20 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
...
+### Eclipse OpenJ9 version 0.30.0 released
 
-We're pleased to announce the availability of Eclipse OpenJ9 v0.29.0 and v0.29.1.
+We're pleased to announce the availability of Eclipse OpenJ9 v0.30.0.
 
-- The 0.29.0 release supports OpenJDK 8 and 11. The 0.29.1 release supports OpenJDK 17. For more information about supported platforms and OpenJDK versions, see [Supported environments](https://www.eclipse.org/openj9/docs/openj9_support/).
+- The 0.30.0 release supports OpenJDK 8, 11 and 17. For more information about supported platforms and OpenJDK versions, see [Supported environments](https://www.eclipse.org/openj9/docs/openj9_support/).
 
-- In this release, Linux AArch64 is a fully supported, production-ready target. JITServer technology is also fully supported on Linux on x86 and Linux on IBM Power systems (64-bit only).
+- In this release, user's should remove obsolete shared cache files created by older releases, ignored options are listed in the javacore, and a new option `-XX:[+|-]EnsureHashed` is added.
```
Comment (mpirvu) [inline news-page-content/latestRelease.md:31]: "user's" --> "users"

Comment (pshipton) [reply news-page-content/latestRelease.md]: updated

**Review by mpirvu — APPROVED** [review]

**Comment by vijaysun-omr** [comment]
@tajila do we need to mention the new command line option that allows for optimization of `identityHashCode` ?

**Comment by tajila** [comment]
@vijaysun-omr The new option `-XX:[+|-]EnsureHashed` is mentioned on line 31

**Comment by pshipton** [comment]
Is someone willing to merge this so the website can show the latest 0.30 release? We can still tweak it afterwards if we come up with something else to add.

**Pull request merged**